apps/mongodb/deployers/mongodb_vars.yaml

```diff
@@ -0,0 +1,5 @@
+# Test-specific parametres
+
+mongodb_deployment: mongo-statefulset.yaml
+namespace: litmus
+test_name: Mongodb-Deployment
```

Comment:
@shashank855 Can you change the test name (Invalid value: \"Mongodb-Deployment\": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character)